threadstacks/signal_handler.cc

```diff
@@ -472,8 +478,11 @@ auto StackTraceCollector::Collect(std::string* error) -> std::vector<Result> {
 std::string StackTraceCollector::ToPrettyString(const std::vector<Result>& r) {
   std::ostringstream ss;
   for (const auto& e : r) {
+    if (e.empty()) {
```

Comment:
Shouldn't this be e.tids.empty()? Also, in this case, let's populate ss with "No threads".